.github/workflows/publish.yml

```diff
@@ -20,6 +20,7 @@ jobs:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@v1
     if: (github.event_name == 'push' || github.event_name == 'pull_request')
     with:
+      toxdeps: "'tox<4'"
```

Comment:
This didn't work? https://github.com/astropy/astropy/actions/runs/3643006172

Reply:
Oh this doesn't need to be here.

```suggestion
```